errai-codegen/src/main/java/org/jboss/errai/codegen/meta/RuntimeAnnotation.java

```diff
@@ -63,4 +74,21 @@ public MetaClass annotationType() {
   public Annotation getAnnotation() {
     return annotation;
   }
+
+  @Override
+  public Map<String, Object> values() {
+    return Arrays.stream(annotation.annotationType().getDeclaredMethods())
+            .filter(m -> !m.isAnnotationPresent(Nonbinding.class))
```

Comment:
Hmm, this code being here might be an issue. NonBinding is a CDI annotation, meaning that an attribute is not used to compare annotations for equality between observer, beans, and injection sites. In general, there could be places that care about NonBinding attributes.

Reply:
Hmm, I see. I'll place this logic of filtering out @Nonbinding methods outside the implementation of MetaAnnotations.